migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/DefaultMigrationTarget.java

```diff
@@ -191,7 +191,7 @@ private void saveSample(Sample sample) throws IOException {
         if (detailed.getParent().getSampleClass() == null && detailed.getParent().getPreMigrationId() != null) {
           // find previously-migrated parent
           detailed.setParent((DetailedSample) serviceManager.getSampleDao().getByPreMigrationId(detailed.getParent().getPreMigrationId()));
-          if (detailed.getParent() == null) {
+          if (detailed.getParent() != null) {
```

Comment:
Was correct before the change. This will make it fail whenever a valid pre-migrated parent is found